src/pages/publication/submission_tab/submission_files_licenses.tsx

```diff
@@ -3,21 +3,29 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { FormikProps, useFormikContext } from 'formik';
 import { Publication } from '../../../types/publication.types';
-import SubmissionContentText from './submission_content_text';
+import FormCardLabel from '../../../components/FormCard/FormCardLabel';
 
 const SubmissionFilesAndLicenses: React.FC = () => {
   const { t } = useTranslation('publication');
   const { values }: FormikProps<Publication> = useFormikContext();
 
   return (
     <>
-      <LabelContentRow label={t('files_and_license.files')}>
-        {values.files.map(file => (
-          <SubmissionContentText>
-            {file.name}({file.license})
-          </SubmissionContentText>
-        ))}
-      </LabelContentRow>
+      <FormCardLabel> {t('files_and_license.files')}</FormCardLabel>
+      {values.files.map(file => (
+        <div key={file.id}>
```

Comment:
en grunn til å ha div her? Hvis ikke kan vi like greit bruke `Fragment`. Var borti akkurat det selv en dag nylig, hvor jeg ikke fikk lov til å bruke `<>` med key, men fikk lov med `<Fragment>`

Reply:
det var pga key, ja. nice

Reply:
Det burde vel kanskje vært en eller annen container for stilingen sin del, men stiling har vi jo ikke,så Fragment it is